v2.0/demo-json-support.md

```diff
@@ -0,0 +1,186 @@
+~~~ sql
+> GRANT ALL ON DATABASE jsonb_test TO maxroach
+~~~
+
+## Step 5. Create a table
+
+In the [built-in SQL client](use-the-built-in-sql-client.html), create a table called `programming`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE rprogramming (
+    id UUID DEFAULT uuid_v4()::UUID PRIMARY KEY,
+    posts JSONB
+  );
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CREATE TABLE programming;
+~~~
+~~~
++--------------+-------------------------------------------------+
+|    Table     |                   CreateTable                   |
++--------------+-------------------------------------------------+
+| programming  | CREATE TABLE programming (                      |
+|              |     id UUID NOT NULL DEFAULT uuid_v4()::UUID,   |
+|              |     posts JSON NULL,                            |
+|              |     CONSTRAINT "primary" PRIMARY KEY (id ASC),  |
+|              |     FAMILY "primary" (id, posts)                |
+|              | )                                               |
++--------------+-------------------------------------------------+
+~~~
+
+## Step 6. Run the code
+
+Now that you have a database, user, and a table, let's run code to insert rows into the table.
+
+The code queries the [Reddit API](https://www.reddit.com/dev/api/) for posts in [/r/programming](https://www.reddit.com/r/programming/). The Reddit API only returns 25 results per page; however, each page returns an `"after"` string that tells you how to get the next page. Therefore, the program does the following in a loop:
+
+1. Makes a request to the API
+2. Inserts the results into the table and grabs the `"after"` string
+3. Uses the new `"after"` string as the basis for the next request
+
+Download the <a href="https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/json/json-sample.go" download><code>json-sample.go</code></a> file, or create the file yourself and copy the code into it:
+
+{% include copy-clipboard.html %}
+~~~ go
+{% include json/json-sample.go %}
+~~~
+
+From a new terminal window, navigate to your sample code file and run it:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ go run json_sample.go
+~~~
+
+The program will take a few minutes to insert rows into your table. If you want to verify that rows of data were inserted into your table,  use the [built-in SQL client](use-the-built-in-sql-client.html):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT count(*) FROM programming;
+~~~
+~~~
++-------+
+| count |
++-------+
+|  7500 |
++-------+
+~~~
+
+## Step 7. Query the data
+
+Once there is data in your table, you can query the data. For this example, retrieve all the entries where the link is pointing to somewhere on YouTube. Specifically, all the entries where `object.data.domain` is `"youtube.com"`. To do that, run:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT id FROM programming WHERE posts @> '{"data": {"domain": "youtube.com"}}';
+~~~
+~~~
+(334 rows)
```

Comment:
It's weird to show only the row count and time. Let's show like 10 rows and then indicate it's been truncated:

```
+--------------------------------------+
|                  id                  |
+--------------------------------------+
| 0036d489-3fe3-46ec-8219-2eaee151af4b |
| 00538c2f-592f-436a-866f-d69b58e842b6 |
| 00aff68c-3867-4dfe-82b3-2a27262d5059 |
| 00cc3d4d-a8dd-4c9a-a732-00ed40e542b0 |
| 00ecd1dd-4d22-4af6-ac1c-1f07f3eba42b |
| 012de443-c7bf-461a-b563-925d34d1f996 |
| 014c0ac8-4b4e-4283-9722-1dd6c780f7a6 |
| 017bfb8b-008e-4df2-90e4-61573e3a3f62 |
| 0271741e-3f2a-4311-b57f-a75e5cc49b61 |
| 02f31c61-66a7-41ba-854e-1ece0736f06b |
| 035f31a1-b695-46be-8b22-469e8e755a50 |
| 03bd9793-7b1b-4f55-8cdd-99d18d6cb3ea |
| 03e0b1b4-42c3-4121-bda9-65bcb22dcf72 |
| 0453bc77-4349-4136-9b02-3a6353ea155e |
...
+--------------------------------------+
(600 rows)

Time: 287.924529ms
```